app/uk/gov/hmrc/exports/controllers/HeaderValidator.scala

```diff
@@ -82,14 +82,30 @@ class HeaderValidator {
     }
   }
 
-  def validateAndExtractNotificationHeaders(
+  def validateAndExtractMovementNotificationHeaders(
     headers: Map[String, String]
-  ): Either[ErrorResponse, ValidatedHeadersNotificationApiRequest] = {
+  ): Either[ErrorResponse, MovementNotificationApiRequest] = {
     val result = for {
       eori <- extractEoriHeader(headers)
       authToken <- extractAuthTokenHeader(headers)
       conversationId <- extractConversationIdHeader(headers)
-    } yield ValidatedHeadersNotificationApiRequest(AuthToken(authToken), ConversationId(conversationId), Eori(eori))
+    } yield MovementNotificationApiRequest(AuthToken(authToken), ConversationId(conversationId), Eori(eori))
+    result match {
+      case Some(vhnar) => Right(vhnar)
```

Comment:
Based on the way how you create a name for it, here you should have `mnar`, not `vhnar`

Reply:
named them all request for ease